py-rse/solutions.Rmd

```diff
@@ -693,6 +704,26 @@ FIXME
 
 FIXME
 
+### Exercise \@ref(teams-ex-contributing)
+
+Your `CONTRIBUTING.md` file might look something like the following:
+
+```text
+# Contributing
+
+Thank you for your interest in contributing to the Zipf's Law package!
+
+If you are new to the package and/or collaborative code development on GitHub,
+feel free to discuss any suggested changes via issue or email.
+We can then walk you through the pull request process if need be.
+As the project grows,
+we intend to develop more detailed guidelines for submitting
+bug reports and feature requests.
+
+We also have a code of conduct (see `CONDUCT.md`).
```

Comment:
```suggestion
We also have a code of conduct (see [`CONDUCT.md`](CONDUCT.md)).
```

Just so it links directly to the file, rather than having to find it.